G1ANT.Addon.UI/G1ANT.Addon.UI/Api/XPathUIElementBuilder.cs

```diff
@@ -82,6 +87,7 @@ protected AutomationElement FindDescendantOrSelf(AutomationElement elem, Compare
         public AutomationElement Root { get; } = AutomationElement.RootElement;
         public XPathUIElementBuilder(AutomationElement root = null)
         {
+            ControlType.Button.GetType();
```

Comment:
why is that needed?

Reply:
Microsoft fucked up initialization of static ControlType.
You need to use it at least use it once to initialize class.

Reply:
@WiktoriaPrusik this is well known bug in UIAutomation, we need to call static constructor of ControlType on our own to allow this class instantation

Reply:
ok, thanks 👍